swagger.yaml

```diff
@@ -2111,7 +2111,19 @@ components:
           type: string
           format: date
         project:
-          $ref: "#/components/schemas/PhaseProjectSubModel"
+          type: object
+          description: A reference to Project Sub Module in Phase Object
```

Comment:
We should remove any mention of sub model/ sub module in descriptions if we are no longer having submodel schemas

Reply:
This applies to other descriptions also